docs/UserGuide.md

```diff
@@ -311,9 +338,9 @@ Removes a class from the module
 
 Format:
 * `delete_class INDEX [tn/TUTORIAL_NAME]`
```

Comment:
similar to above! as in the notes section `[PARAM]` is indicated as optional so this may make users misunderstand that INDEX is compulsory while tn/ is optional